app/js/views/app_view.js

```diff
@@ -1,20 +1,53 @@
 define([
   'jquery',
   'backbone',
-  'text!templates/app_view.ejs'
-], function ($, Backbone, indexTemplate) {
+  'text!templates/app_view.ejs',
+  'collections/todo_item_collection',
+  'models/todo_item',
+  'views/todo_item_list_view'
+], function ($, Backbone, indexTemplate, TodoItemCollection, TodoItem, TodoListView) {
   return Backbone.View.extend({
     el: this.$('#todo-app'),
 
+    initialize: function(){
+      this.todoItemCollection = new TodoItemCollection();
+      this.todoListView = new TodoListView({todoItemCollection: this.todoItemCollection});
+    },
+
+    events:{
+      "click #add-todo-button" : "addTodo",
+      "keydown #todo-title-textbox" : "onKeyDownEventHandler"
+    },
+
+    onKeyDownEventHandler: function(e){
+      if(e.keyCode==13){
+        this.$('#add-todo-button').click();
+      }
+    },
+
     currentDate: function () {
       return new Date();
     },
 
+    addTodo: function() {
+      var todoTitle = this.$('#todo-title-textbox').val();
+      if(!todoTitle) return;
+      var todoItem = new TodoItem({
+        title: todoTitle,
```

Comment:
We need to fix linting problems. You can find how to run linting on repository documentation.